evm/src/cpu/kernel/asm/core/call.asm

```diff
@@ -0,0 +1,19 @@
+// Handlers for call-like operations, namely CALL, STATICCALL and DELEGATECALL.
+
+global call:
+    // TODO: Transfer value
+    %jump(call_common)
+
+global staticcall:
+    // TODO: Set flag to indicate
+    %jump(call_common)
+
+global delegatecall:
+    // TODO:
+    %jump(call_common)
+
+call_common:
+    %create_context
```

Comment:
I had a look at this and at `create_context` and this is indeed what I was imagining for contract calls.